CHANGELOG.rst

```diff
@@ -1,3 +1,12 @@
+**2.0.0 - 08/13/24**
+
+ - Implement Vivarium's new results processing
```

Comment:
I think you should use the breaking/major/minor distinctions here as well. Breaking would be results processing, major is simulation backups, and everything else is minor.